Scrapper-Service/test/mongodb_test.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import json
+import warnings
+from datamodels import Conference
+from datamodels import Metadata
+from database  import mdb
+from datetime import datetime
+from logging import Logger
+
+class MongoDbTestCase(unittest.TestCase):
+#    def __init__(self, methodName):
+#        super().__init__(methodName)
+    
+    def cvt(self,dt):
```

Comment:
* Can you resort to some basic Conference object, so that it's easier to read the unit test.

some thing like Conference("name", datetime.datetime.now(), "url.com" , Metadata)

It's much more easier to trace behaviour.

Reply:
Last Time, I have done this but you told me that was wrong.